test/utilities.js

```diff
@@ -1,146 +0,0 @@
-/**
- * funds accounts with specified value for Mintable Token
- * The object consists of:
- * 1.) minter/owner of token (i.e. minter)
- * 2.) list of accounts
- * 3.) ERC20Mintable token
- * 4.) amount to be funded
- */
-const fundAccounts = async function (minter, accounts, token, amount) {
```

Comment:
A lot of these methods seem to have been removed - removing unused code is awesome! Two :bird: one stone.

Reply:
I believe the `fundAccounts` was a helper method for snapp contracts, but I can imagine this would also be useful for StableX in the exact same way. Perhaps we should double check on this before removing?